code/modules/clothing/outfits/plasmaman.dm

```diff
@@ -235,3 +235,35 @@
 	head = /obj/item/clothing/head/helmet/space/plasmaman/centcom_intern
 	uniform = /obj/item/clothing/under/plasmaman/centcom_intern
 	gloves = /obj/item/clothing/gloves/color/plasmaman/centcom_intern
+
+/datum/outfit/plasmaman/party_janitor
+	name = "ERP Cleaning Plasmaman"
+
+	mask = /obj/item/clothing/mask/breath
+	head = /obj/item/clothing/head/helmet/space/plasmaman/janitor
+	uniform = /obj/item/clothing/under/plasmaman/janitor
+	gloves = /obj/item/clothing/gloves/color/plasmaman/janny
+
+/datum/outfit/plasmaman/party_bouncer
+	name = "ERP Bouncer Plasmaman"
```

Comment:
🤔

Reply:
Not sure if you are pointing out the name... I'm just following the human version
>/datum/outfit/centcom/ert/security/party
name = "ERP Bouncer"